FRCore/FRCore/Network/FRPinningHandler.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FRPinningHandler.swift
+//  FRCore
+//
+//  Copyright (c) 2022 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+
+@objc
+public protocol FRPinningHandlerProtocol: URLSessionTaskDelegate {
```

Comment:
Let's discuss this, as the SDK was already using the URLSessionTaskDelegate, hence I left it for compatibility. You might right though and this might not be needed